package/src/ssr/ApolloNextAppProvider.tsx

```diff
@@ -43,6 +43,8 @@ export const ApolloNextAppProvider = ({
   return (
     <_ApolloProvider
       client={clientRef.current}
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
```

Comment:
This is just to get the CI to shut up until #61 is merged - I'll remove that later.